src/core_modules/capture-core/components/Section/Section.component.js

```diff
@@ -60,6 +61,7 @@ class Section extends Component<Props> {
         return (
             <div>
                 {this.getHeader()}
```

Comment:
this.props.header is processed in the function getHeader(). I haven't understood why yet, and thus don't know if such a function is needed for this.props.description.

Reply:
This component should totally be refactored, but let's leave that for another time. What you did here makes perfect sense 👍